template_package/etc/bash_completion.d/mdtool_completion.sh

```diff
@@ -52,18 +52,22 @@ _mdtool_complete()
 
 	elif [ $COMP_CWORD -eq 4 ]; then
 		if [[ "$prev2" == "motor" ]]; then
+			if [[ "$cur" == ./* || "$cur" == /* ]]; then
```

Comment:
if rather easy to do, autocomplete only files with .cfg extension. Now it hints all the filenames.

Reply:
But we want to autocomplete also directories, right? But I'll take a look at it

Reply:
only dirs + .cfg, as we only have .cfg motor configs, and .ini is mdtool config